File: Python/8py 2/remove_marks4.py
"""
|--------------------------------------------------------------------------
| Exclamation marks series #4: Remove all exclamation marks from sentence but ensure a exclamation mark at the end of string
|--------------------------------------------------------------------------
|
"""

def remove(s):
   return s.replace("!", "") + "!"


def remove(s):
   return '{}!'.format(s.replace('!', ''))


def remove(s):
   return f'{s.replace("!", "")}!'


print(remove("Hi!"), "Hi!")
print(remove("Hi!!!"), "Hi!")
print(remove("!Hi"), "Hi!")
print(remove("!Hi!"), "Hi!")
print(remove("Hi! Hi!"), "Hi Hi!")
print(remove("Hi"), "Hi!")

File: JavaScript/8js 3/remove_marks4.js
/*
|--------------------------------------------------------------------------
| Exclamation marks series #4: Remove all exclamation marks from sentence but ensure a exclamation mark at the end of string
|--------------------------------------------------------------------------
|
*/

function remove(string) {
   return string.replace(/!/g, "") + "!";
}


const remove = (s) => `${s.replace(/!/g, "")}!`;


function remove(string) {
   return string.split("!").join("") + "!";
}


console.log(remove("Hi!"), "Hi!");
console.log(remove("Hi!!!"), "Hi!");
console.log(remove("!Hi"), "Hi!");
console.log(remove("!Hi!"), "Hi!");
console.log(remove("Hi! Hi!"), "Hi Hi!");
console.log(remove("Hi"), "Hi!");